server/src/main/java/com/cloud/api/ApiServlet.java

```diff
@@ -186,22 +193,30 @@ void processRequestInContext(final HttpServletRequest req, final HttpServletResp
         }
 
         try {
-
-            if (HttpUtils.RESPONSE_TYPE_JSON.equalsIgnoreCase(responseType)) {
-                resp.setContentType(ApiServer.JSONcontentType.value());
-            } else if (HttpUtils.RESPONSE_TYPE_XML.equalsIgnoreCase(responseType)){
-                resp.setContentType(HttpUtils.XML_CONTENT_TYPE);
-            }
+            resp.setContentType(HttpUtils.XML_CONTENT_TYPE);
```

Comment:
@DaanHoogland should we force xml?

Reply:
It wouldn´t matter in the existing code as line 179 (former version 168) https://github.com/apache/cloudstack/pull/6393/files#diff-cc92beff292fdad7835437d622159db6e1ae90d0ebac7fb4ae73c813faa8896eR179 hard-codes the `reponseType` to `HttpUtils.RESPONSE_TYPE_XML`.  The second branch would always be executed.